modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/SqlQueryProcessor.java

```diff
@@ -596,14 +624,22 @@ private CompletableFuture<AsyncSqlCursor<InternalSqlRow>> queryScript(
     ) {
         String schemaName = properties.get(QueryProperty.DEFAULT_SCHEMA);
         ZoneId timeZoneId = properties.get(QueryProperty.TIME_ZONE_ID);
+        Long queryTimeout = properties.getOrDefault(QueryProperty.QUERY_TIMEOUT, 0L);
```

Comment:
the same about default and Integer.MAX_VALUE below

Reply:
Fixed.